extensions/amp-ad-network-doubleclick-impl/0.1/test/test-doubleclick-a4a-config.js

```diff
@@ -56,33 +58,32 @@ describe('doubleclick-a4a-config', () => {
   });
 
   describe('#doubleclickIsA4AEnabled', () => {
-    it('should enable a4a when requested and on CDN', () => {
-      mockWin.location = parseUrl(
-          'https://cdn.ampproject.org/some/path/to/content.html');
+    it('should enable a4a when native crypto is supported', () => {
+      forceExperimentBranch(mockWin, DFP_CANONICAL_FF_EXPERIMENT_NAME,
```

Comment:
Ideally we wouldn't force anymore.  Instead we would setup stub for experiment selection function and get experiment functions returning the values we want combined withArgs

Reply:
Why is it better to stub than to force the experiment branch? Seems like it would be a more useful test for catching changes if we call the actual experiment selection

Reply:
Agreed, see my previous comment that we do not currently test experiment selection at all.